.github/ISSUE_TEMPLATE/implement-new-feature.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Implement new feature
+about: Create a new featrue
+title: ''
+labels: ''
```

Comment:
`enhancement`?

Reply:
Oh, we didn't decide the label templates, so I didn't set it.

Reply:
I see! I am okay with the current labels. You can suggest new labels or any modifications.